chore: move Python Solver to new repo

Christopher-Chianelli opened the pull request:
timefold-solver PR: https://github.com/TimefoldAI/timefold-solver/pull/1693

Christopher-Chianelli committed:
chore: move Python packages back to their own repo

Christopher-Chianelli committed:
chore: move Python docs to new repo

triceo commented on README.md:
```diff
@@ -0,0 +1,188 @@
+_Planning optimization made easy._  
+[timefold.ai](https://timefold.ai)
+
+[![GitHub Discussions](https://img.shields.io/github/discussions/TimefoldAI/timefold-solver?style=for-the-badge&logo=github)](https://github.com/TimefoldAI/timefold-solver/discussions)
```
Let's remove this line.

triceo commented on README.md:
```diff
@@ -0,0 +1,188 @@
+[![Python support](https://img.shields.io/badge/Python-3.10+-brightgreen.svg?style=for-the-badge)](https://www.python.org/downloads)
+[![License](https://img.shields.io/github/license/TimefoldAI/timefold-solver?style=for-the-badge&logo=apache)](https://www.apache.org/licenses/LICENSE-2.0)
+
+[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=ai.timefold:timefold-solver&metric=reliability_rating)](https://sonarcloud.io/summary/new_code?id=ai.timefold:timefold-solver)
+[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=ai.timefold:timefold-solver&metric=security_rating)](https://sonarcloud.io/summary/new_code?id=ai.timefold:timefold-solver)
+[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=ai.timefold:timefold-solver&metric=sqale_rating)](https://sonarcloud.io/summary/new_code?id=ai.timefold:timefold-solver)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=ai.timefold:timefold-solver&metric=coverage)](https://sonarcloud.io/summary/new_code?id=ai.timefold:timefold-solver)
```
This will no longer work.

triceo commented on README.md:
```diff
@@ -0,0 +1,188 @@
+
+## Requirements
+
+- [Install Python 3.10 or later.](https://www.python.org)
```
Maybe we list the Python versions specifically, considering that 3.13 doesn't work?

triceo commented on .mvn/wrapper/maven-wrapper.properties:
```diff
@@ -0,0 +1,18 @@
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.1/apache-maven-3.9.1-bin.zip
+wrapperUrl=https://repo.maven.apache.org/maven2/org/apache/maven/wrapper/maven-wrapper/3.2.0/maven-wrapper-3.2.0.jar
```
Latest, please?

Christopher-Chianelli replied:
This is a copy from timefold-solver

triceo replied:
And if I noticed it there, I'd have fixed it there too. :-)

Christopher-Chianelli committed:
chore: review comments

- use `mvn -N wrapper:wrapper -Dmaven=3.9.10` to generate wrapper
- modfify pom.xml to not include docs module in python build

Christopher-Chianelli committed:
chore: remove docs
